_docs-sources/guides/stay-up-to-date/cis/cis-1.5.0/deployment-walkthrough/step-4-verify-the-code-changes.md

```diff
@@ -0,0 +1,100 @@
+---
+sidebar_label: Verify the code changes
+---
+
+# Step 4: Verify the code changes
+
+:::caution
+
+We strongly recommeded that you verify the changes that have been made before executing Terraform/Terragrunt `apply`
+
+:::
+
+Upgrading to CIS AWS Foundations Benchmark v1.5.0 includes migrating to our new CIS RDS module, this breaking change will result in data loss if not updated correctly.
+
+We strongly recommend that you run `terraform diff` and verify that any RDS resources you have will be "updated in-place" and not destroy and new resources created.
+
+## The migrations were correctly executed
+
+If the migrations have been correctly executed then running `terraform diff` will output a message similar to this one:
+
+    Plan: 0 to add, 6 to change, 0 to destroy.
+
+This indicates that the existing resources will be "updated in-place" Terraform/Terragrunt `apply` is executed.
+
+The output of the diff will contain messages similar to these:
+
+    Terraform will perform the following actions:
+
+    # module.rds.module.rds_alarms.aws_cloudwatch_metric_alarm.rds_disk_space_available[0] will be updated in-place
+    ~ resource "aws_cloudwatch_metric_alarm" "rds_disk_space_available" {
+        ...
+        }
+
+    # module.rds.module.rds_alarms.aws_cloudwatch_metric_alarm.rds_high_cpu_utilization[0] will be updated in-place
+    ~ resource "aws_cloudwatch_metric_alarm" "rds_high_cpu_utilization" {
+        ...
+        }
+
+    ...
+
+    Plan: 0 to add, 6 to change, 0 to destroy.
+
+
+## The migrations were not correctly executed
```

Comment:
Another thing to check: if `tflint` is enforcing SG rules correctly.

Reply:
I updated the text around tflint, in the `step-2` page:
> The CIS AWS Foundations Benchmark v1.5.0 includes a requirement to "Ensure no security groups allow ingress from ::/0 to remote server administration ports", Patcher also includes a patch that configures `tflint` with the [gruntwork-io/tflint-ruleset-aws-cis](https://github.com/gruntwork-patcher-dev/tflint-ruleset-aws-cis) to scan your
infrastructure and enforce this requirement. [Terragrunt now supports running `tflint` as a `before_hooks`](https://terragrunt.gruntwork.io/docs/features/hooks/#tflint-hook).